SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -30,69 +35,66 @@ bool WriteFlagSection(TSharedRef<TJsonWriter<>> Writer, const FString& Key, cons
 	return true;
 }
 
-bool WriteWorkerSection(TSharedRef<TJsonWriter<>> Writer, const FWorkerTypeLaunchSection& Worker)
+bool WriteWorkerSection(TSharedRef<TJsonWriter<>> Writer, const FName& WorkerTypeName, const FWorkerTypeLaunchSection& WorkerConfig)
 {
 	Writer->WriteObjectStart();
-		Writer->WriteValue(TEXT("worker_type"), *Worker.WorkerTypeName.ToString());
+		Writer->WriteValue(TEXT("worker_type"), *WorkerTypeName.ToString());
 		Writer->WriteArrayStart(TEXT("flags"));
-			for (const auto& Flag : Worker.Flags)
+			for (const auto& Flag : WorkerConfig.Flags)
```

Comment:
The `Flags` section appears to be both an array and a map at the same time. It should only be a map. This is causing double icons to show:
![image](https://user-images.githubusercontent.com/31517089/80955934-17dd3400-8df8-11ea-9501-6fcb4b823ab5.png)